feature/experimental/p4rt/ate_tests/performance_test/performance_test.go

```diff
@@ -0,0 +1,955 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package performance_test
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"sort"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/internal/p4rtutils"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+var (
+	p4InfoFile                              = flag.String("p4info_file_location", "../../wbb.p4info.pb.txt", "Path to the p4info file.")
+	streamName                              = "p4rt"
+	gdpInLayers         layers.EthernetType = 0x6007
+	lldpInLayers        layers.EthernetType = 0x88cc
+	lldpEthertype                           = uint32(0x88cc)
+	gdpEtherType                            = uint32(0x6007)
+	deviceID                                = *ygot.Uint64(1)
+	portID                                  = *ygot.Uint32(10)
+	electionID                              = *ygot.Uint64(100)
+	metadataIngressPort                     = *ygot.Uint32(1)
+	metadataEgressPort                      = *ygot.Uint32(2)
+
+	Duration        = uint32(20)
+	gdpBitRate      = uint64(200000)
+	lldpBitRate     = uint64(100000)
+	trPacketRate    = uint64(324)
+	packetinPktsize = uint32(300)
+	srcMac          = flag.String("srcMac", "00:01:00:02:00:03", "source MAC address for PacketIn")
+	lldpDstmac      = flag.String("lldpDstmac", "01:80:c2:00:00:0e", "source MAC address for PacketIn")
+	gdpDstmac       = flag.String("gdpDstmac", "00:0a:da:f0:f0:f0", "source MAC address for PacketIn")
+	TTL1            = uint8(1)
```

Comment:
I don't think this needs to be exported

Reply:
Is this for the src, dst MAC or TTL?

Reply:
Of those, TTL is the only one that you've exported (capitalized) here.